applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -198,7 +201,7 @@ def MarkKuttaElements(self):
                 elem.SetValue(CPFApp.KUTTA, True)
 
     @staticmethod
-    def CheckIfElemIsCutByWake(elem):
+    def __CheckIfElemIsCutByWake(elem):
         nneg=0
         distances = elem.GetValue(KratosMultiphysics.ELEMENTAL_DISTANCES)
```

Comment:
Note that in here you are retrieving the discontinuous elemental distances, which might not coincide with the nodal ones if you call this method after using the modify distance process.

Reply:
In this specific case we are not using the modified_distance process since we are analytically computing the distances here (note that is the distance function that we use for the wake, which is different from the distance funcion that we will use for the embedded case):
https://github.com/KratosMultiphysics/Kratos/blob/529adfa67c2859e68711846eb6dd977374822eb9/applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py#L146

Reply:
OK. But how are you computing the distances? By means of the `CalculateDistanceToSkinProcess` process? If this is the case, the nodal values do not strictly coincide with the elemental ones.

Reply:
No, we are not using the CalculateDistanceToSkinProcess!
What I meant is that we are using a custom ComputeDistanceToWake method (this is 2D).

In any case, thanks for pointing it out because for the 3D we will have to use the CalculateDistanceToSkinProcess and your point may be important then.

Reply:
Then I suggest you to leave a comment in the current version as a reminder.